tensorflow/python/keras/layers/convolutional.py

```diff
@@ -72,6 +72,10 @@ class Conv(Layer):
       Specifying any stride value != 1 is incompatible with specifying
       any `dilation_rate` value != 1.
     padding: One of `"valid"`,  `"same"`, or `"causal"` (case-insensitive).
+      `"valid"` means no padding. `"same"` results in padding evenly to 
+      the left/right or up/down of the input such that output has the same 
+      height/width dimension as the input. `"causal"` results in causal 
+      (dilated) convolutions, e.g. `output[t]` does not depend on `input[t+1:]`.
```

Comment:
you might want to replicate the `causal` part for the rest of the PR

Reply:
I'm not sure if we have `"causal"` for Conv2D and Conv3D. We should instead use the `dilation_rate` parameter.

Reply:
We can't get the same causal dilations with only the `dilation_rate` parameter, unless the `causal` padding variant is implemented for that layer. Therefore we can resolve this thread